### DISCUSSION TITLE
different between web-ui FLUX generation

### DISCUSSION THREAD

**Discussion by kalle07** [mcmonkeyprojects/SwarmUI#364] [Q&A]
What are the main differences between FLUX on webui ?

copy of metadata form webui - image
Steps: 25, Sampler: Euler, Schedule type: SGM Uniform, CFG scale: 1.0, Distilled CFG Scale: 3.5, Seed: 2405129555, Size: 680x1024, Model hash: 0c6631c392, Model: !flux1DevFp8_v10, Version: f2.0.1v1.10.1-previous-586-gd4d8ad40, Diffusion in Low Bits: Automatic (fp16 LoRA), Module 1: t5xxl_fp16, Module 2: ae, Module 3: ViT-L-14-BEST-smoot

i take the same parameter (oc same prompt) in swarmui and get i different image and not only a small part of the image like a small  blurry object in background...
Flux Guidance Scale samle like "Distilled CFG Scale: 3.5" ?!?

i do not found any typ of "ae" Module2 ... and if i dont have it on webui it dont work
https://huggingface.co/black-forest-labs/FLUX.1-dev/tree/main
file 
ae.safetensors

iv tryed the clip_l and t5xxl_enconly and my too but thats not the point ...

**Comment by mcmonkey4eva** [comment]
what

**Reply by kalle07** [reply]
what ?
your questions are helpful   : D

webui 
https://github.com/lllyasviel/stable-diffusion-webui-forge

where you apply the  "ae.safetensors"  ?
that makes the differents not clip or t5xxl

**Reply by mcmonkey4eva** [reply]
the name for the thing you linked is "Forge"
The word "webui" is shorthand for "Web User Interface" and is a general description of Forge, Auto1111, ComfyUI, Swarm, and every other SD UI and so many other non-SD-related interfaces

You don't need to download the AE, Swarm automatically downloads and applies it for you. It doesn't make any visible difference, if the AE is missing it just doesn't work. You can manually select it (under the VAE tab) if you really want, but there's no reason to, flux only has only valid AE.

`Flux Guidance Scale samle like "Distilled CFG Scale: 3.5" ?!?` yes, "Distilled CFG" is just a mislabeling in Forge of the same parameter.

`what ?
your questions are helpful : D`
Your posts are all written incoherently. Please take the time to write clearly and understandably if you expect someone else to take the time to figure out what you mean and answer you

**Reply by kalle07** [reply]
difficult to describe something that I don't really understand ;)

but the AE is the point I already had this file, but 
your VAE standard is
t5xxl_fp8_e4m3fn

if a choose
AE

error:
- Request requires model 'ae-f16.gguf' but the backend does not have that model

but my flux model is´nt a gguf, its the original DEV_fp8 (11GB)

yes i have "ae-f16.gguf" and the image is generating but in the end
- ComfyUI execution error: 'VAE' object has no attribute 'vae_dtype'

this is my AE model
https://huggingface.co/black-forest-labs/FLUX.1-dev/blob/main/ae.safetensors
In my opinion, this is what makes the difference between my pictures in Forge and Swarm

**Reply by kalle07** [reply]
here looks like this
![grafik](https://github.com/user-attachments/assets/7e4d79d1-5428-4c48-9ede-77eacf789f4d)

and here after generation
![grafik](https://github.com/user-attachments/assets/6374fe2f-e3cb-4019-9fca-44a823107300)

why there is the "vae:ae" but in the menue is another one ?

**Reply by mcmonkey4eva** [reply]
Get that T5 model out of your VAE folder, T5 is not a VAE.

**Comment by kalle07** [comment]
why you close a disussion i mean some one else can answer it  ^^
